wordpress-mobile/WordPress-Android
# Redesign the header in the screen of comment details

Part of https://github.com/Automattic/wordpress-mobile/issues/30
Designs: yWt5gg3nWORhu079Qfv3mS-fi-1135_4600

This PR updates the header in the screen of comment details to the latest design.

| Before | After |
|--|--|
| ![Screenshot_20240503-095415 (1)](https://github.com/wordpress-mobile/WordPress-Android/assets/3839951/6b8978b3-3ab5-41ea-8faf-deb55ea2f0a7) | ![Screenshot_20240503-095415 (1)](https://github.com/wordpress-mobile/WordPress-Android/assets/3839951/9d4b867c-0907-4951-848e-15a380a2ca22) |
| ![Screenshot_20240503-095415 (1)](https://github.com/wordpress-mobile/WordPress-Android/assets/3839951/72ab0ba6-45be-4641-bb7a-d8f37f2515fd) | ![Screenshot_20240503-095415](https://github.com/wordpress-mobile/WordPress-Android/assets/3839951/1e7d3260-3528-453c-a468-17fe948925be) |

-----

## To Test:

1. Sign in the JP app
2. Go to the tab of `Notifications`
3. Find a notification of comment and click on it
4. There **should** be a header at the top of the screen
5. There **should not** be a title on the Toolbar
6. Click on the header.
7. The behavior should be the same as before.
8. Go to My Site -> Comments
9. Click on a comment in that comment list
10. Repeat step 4-7
11. Done, thank you!

P.S. Those design related to avatar, user name will be handled in https://github.com/Automattic/wordpress-mobile/issues/31

**Known existing issue**
Currently, we navigate users to `ReaderCommentListActivity` if we click on a comment notification for a comment. See p1714733619848769/1714162403.634609-slack-C06BWNSR02K


-----

## Regression Notes

1. Potential unintended areas of impact

    - Comments, Notifications

12. What I did to test those areas of impact (or what existing automated tests I relied on)

    - Manual

13. What automated tests I added (or what prevented me from doing so)

    - N/A

-----

## PR Submission Checklist:

Skipped

-----

## Testing Checklist (strike-out the not-applying and unnecessary ones):
